diamm/models/site/contribution.py

```diff
@@ -33,4 +33,4 @@ class Meta:
                                max_length=255, blank=True, null=True)
 
     created = models.DateTimeField(auto_now_add=True)
-    updated = models.DateTimeField(auto_now=True)
+    updated = models.DateField(auto_now=True)
```

Comment:
This should remain a `DateTimeField` since we will want to maintain the time that something was changed, and not just the date.